doc/whats-new.rst

```diff
@@ -64,6 +64,10 @@ Deprecations
 Bug fixes
 ~~~~~~~~~
 
+- Writing to unaligned zarr chunks with ``region`` will now raise an error
```

Comment:
```suggestion
- Partial writes to existing chunks with ``region`` will now raise an error
```

Didn't know what "unaligned" meant here. Does this read better?

Reply:
Thanks, that is better...